setup.py

```diff
@@ -13,11 +13,9 @@
     version = f.read().strip()
 
 install_requires = [
-    'wazimap>=1.2,<1.3',
-    'wagtail>=1.13.1,<1.14',
+    'wazimap>=1.2,<2.0',
```

Comment:
I think this blocks:
```shell
RUN pip install -q git+https://github.com/CodeForAfricaLabs/wazimap.git@master#egg=wazimap
```
since `CodeForAfricaLabs/wazimap` is @ `v2.0` _after_ releases support was merged (OpenUpSA/wazimap is also @ v2.0 for the releases branch).

![screenshot from 2019-01-10 13-48-14](https://user-images.githubusercontent.com/1779590/50966386-d11e4900-14e5-11e9-9216-ef1d4dbb4c8c.png)